website/docs/how-to/how-to-create-service-accounts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to create service accounts
+---
+
+:::info availability
+
+Service accounts is an upcoming enterprise feature, and is scheduled to become available in one of the next few releases.
+
+:::
+
+[Service accounts](../reference/service-accounts.md) enable Unleash admins to create accounts that act as users and respect the same set of permissions, however they do not have a password and cannot log in to the Unleash UI. Instead, they are intended to be used to access the Unleash API programatically.
+
+## Step 1: Navigate to the service accounts page {#step-1}
+
+Navigate to the _Service accounts_ page in the admin UI (available at the URL `/admin/service-accounts`). Use the "Settings Wheel" on the navigation menu and select "Service accounts".
```

Comment:
This should probably be "settings button" instead. There's no non-visual indication that the button presents itself as a wheel, so I'm afraid it could be problematic to users with vision impairment. Plus, a settings wheel might conjure a whole different image.

```suggestion
Navigate to the _service accounts_ page in the admin UI (available at the URL `/admin/service-accounts`). Use the _settings_ button in the navigation menu and select "service accounts".
```